hstyle/blog/js/prism.js

```diff
@@ -1,12 +1,1144 @@
+var _self = "undefined" != typeof window ? window : "undefined" != typeof WorkerGlobalScope && self instanceof WorkerGlobalScope ? self : {},
```

Comment:
what have you changed? It looks similar..

Reply:
I've just formatted the js code to look easier. The code is from prism.js
the code is taken from:
https://prismjs.com/download.html#themes=prism-okaidia&plugins=toolbar+copy-to-clipboard
..

Reply:
I've seen the code.. It looks ok